CHANGES.md

```diff
@@ -53,6 +61,7 @@ release:
 
 <!-- Changes that affect Black's preview style -->
 
+- Add `--unstable` style (#4096)
```

Comment:
Can we add a quick link / blurb here for preview style codebases that want hug_parens?

Reply:
I kept the hugging feature (`hug_parens_with_braces_and_square_brackets`) in preview. Or maybe you're referring to `multiline_string_handling`?

Reply:
Also possibly `hug_parens_with_braces_and_square_brackets` should go into unstable; I haven't thought much about it beyond not putting it in 2024 stable. We have 11 months left to do that though.

Reply:
Ah sorry, I was wrong on the specifics. Just thought more of a mention in changelog could be nice for whatever unstable features users may want

Reply:
Good point, I'll add something.